include/tscore/ink_config.h.cmake.in

```diff
@@ -187,3 +187,5 @@ const int DEFAULT_STACKSIZE = @DEFAULT_STACK_SIZE@;
 #cmakedefine YAMLCPP_LIB_VERSION "@YAMLCPP_LIB_VERSION@"
 
 #cmakedefine01 TS_HAS_CRIPTS
+
+#cmakedefine HAVE_ZSTD_H 1
```

Comment:
I think this should be `TS_USE_ZSTD` for consistency with the other optional features and be grouped with them in this file. @cmcfarlen the other optional feature defines use `#cmakedefine01`, should this one as well?